test/test_examples.py

```diff
@@ -112,8 +112,4 @@ def twenty_one_triangles():
 
 if __name__ == "__main__":
     grid = twenty_one_triangles()
-    print(grid.edges)
```

Comment:
When I put code in an if __name__ block in a test file, it's so you can play around a bit without dealing with the full-on test runner. That's usually through-away code (as it won't be run by the test runner anyway).